packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +57,24 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with {result: "skip"} when there is a skip record': () => {
+        const dns = {
+          ...ERROR_CODES,
```

Comment:
@chenba  I am little bit confused with line 62 and 75

Reply:
`dns` here is used to mock the Nodejs dns module in validate-email-domain.js.  You don't really need it for this test since validate-email-domain.js shouldn't be using `dns` to resolve a domain.